src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1190,3 +1194,88 @@ def run_datamover(self, test_desc=None,
                 self.fail("stderr expected {}: {}".format(s, test_desc))
 
         return result
+
+    def run_dm_activities_with_ior(self, tool, create_dataset=False):
```

Comment:
I had envisioned some "round trip" datamover functions like this before, but just never got around to it! :)
I think it would be a good idea at some point (independently) to change some of the CI datamover tests to use this function. That way as the test base is changed, we won't lose coverage on this function. E.g. if this function is only used in deployment tests and a change breaks it, we won't know until the test fails in deployment

Reply:
Yeah I was thinking of doing the same as part of separate tickets as well.